docs/postgres.md

```diff
@@ -118,6 +118,9 @@ performance:
 Note that the appropriate values for those fields depend on the amount
 of free memory the database host has available.
 
+Additionally, admins of large deployments might want to consider using huge pages
```

Comment:
"huge pages" is a linux kernel feature, so this only applies to deployments running on Linux. Maybe it's worth saying "large deployments on Linux".

(We don't really support running it on anything else AFAIK, so maybe this is a needless distinction to draw. I leave it to your discretion!)